### PR TITLE
add the as-module option

### PR DESCRIPTION
**Pull request by wenkaifan0720**

### CODE REVIEW

File: lib/src/flutterflow_cli_base.dart [hunk 1]
```diff
@@ -55,6 +57,7 @@ Future<String?> exportCode({
   required bool includeAssets,
   required bool unzipToParentFolder,
   required bool fix,
+  required exportAsModule,
```
Comment (aemelyanovff) [inline lib/src/flutterflow_cli_base.dart:60]: Nit: add `bool`.

File: bin/flutterflow_cli.dart [hunk 2]
```diff
@@ -79,6 +80,10 @@ ArgResults _parseArgs(List<String> args) {
           'download all project code directly into the specified directory, '
           'or the current directory if --dest is not set.',
       defaultsTo: true,
+    )
+    ..addFlag(
+      'as-module',
```
Comment (aemelyanovff) [inline bin/flutterflow_cli.dart:85]: Nit: let's make this option negatable, like so users can explicitly set `--no-as-module` if they want. It's not particularly important, but we do it for other boolean flags.

If you do it, please update README.md: `--as-module` -> `--[no-]as-module` (in both places).

**Review by aemelyanovff — APPROVED** [review]
LGTM except a couple of tiny nits.

But please don't merge this until the API is ready to accept the new parameter.

File: bin/flutterflow_cli.dart (hunk 2 above)
Comment (wenkaifan0720) [reply bin/flutterflow_cli.dart:85]: I thought if the user do not have this flag, then we don't export the code as module. But I think it makes sense to follow the rule for other boolean flags.

**Pull request merged**